src/pages/bounty-creator/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { PropsWithChildren, useState } from "react";
+import { Avatar } from "@chakra-ui/react";
+import { DashboardSearch } from "@/components/shared";
+import SidebarMobile from "@/components/Sidebar/mobile";
+import SidebarDesktop from "@/components/Sidebar/desktop";
+import CreatorDashboard from "@/components/creator-bounty/dashboard";
+import { useRouter } from "next/router";
+import AvailableBounty from "@/components/creator-bounty/available-bounty";
+import BountyMembers from "@/components/creator-bounty/bounty-members";
+
+const bountiesNav: { id: number; value: string; path?: string }[] = [
+  { id: 1, value: "Available Bounties" },
+  { id: 2, value: "Members" },
+];
+
+export type customNavPropTypes<
+  T extends readonly { value: string; id: number }[]
+> = {
+  onChange: (value: T[number]["value"]) => void;
+  data: T;
+  active: T[number]["value"];
+  className?: string;
+  type?: 1 | 2;
+  wide?: boolean;
+};
+
+export const CustomNavAlt = <
+  T extends readonly { value: string; id: number; path?: string }[]
+>({
+  onChange,
+  data,
+  active,
+  className,
+  type = 1,
+  wide,
+}: PropsWithChildren<customNavPropTypes<T>>) => {
+  const { push } = useRouter();
+
+  const _handleTabChange = (value: string, path?: string) => {
+    typeof onChange === "function" && onChange(value);
+    path && push(path);
+  };
+
+  const activeText = type === 2 ? "text-white" : "text-dark-grey";
+
+  return (
+    <div className={className}>
+      {data.map(({ value, id, path }) => (
+        <div
+          key={id}
+          className="relative inline-block cursor-pointer group"
+          onClick={() => _handleTabChange(value, path)}
+          data-testid={`project-tab_${value}`}
+        >
+          <span
+            className={`text-center text-input-border text-xs sm:text-sm font-filson-medium mx-2 group-hover:text-accepted mb-[11.58px] mt-[13.5px] inline-block
+							${active === value && [activeText]}
+							${wide && "mx-10"}
+						`}
+          >
+            {value}
+          </span>
+          <div
+            className={`rounded-t-[10px] h-[3.5px] bg-accepted
+							${active !== value && "invisible"}
+						`}
```

Comment:
Please work on the alignment here

Reply:
Please **NOTE:** subsequent PRs should include a note/description of work done since you cannot provide screenshots. This should help me know where to look when reviewing the work on the browser apart from reviewing code changes on GitHub. e.g
This PR contains the following completed task
- Creator Bounty page
- Create/Edit Bounty
- Bounty detail 
- Accept/Reject modal/delete
- Settings page
- View Bounty Component
- Members Component Tab.

Let me know if I missed anything.

Reply:
Overall great job so far. Welldone 👍